kubejs/assets/dustrial_decor/lang/pl_pl.json

```diff
@@ -0,0 +1,11 @@
+{
+  "item.dustrial_decor.cast_iron_billet": "Kęs Żelaza Andezytowego",
+  "block.dustrial_decor.cast_iron_block": "Blok Żelaza Andezytowego",
+  "block.dustrial_decor.cast_iron_pillar": "Filar Żelaza Andezytowego",
+  "block.dustrial_decor.cast_iron_bricks": "Cegły Żelaza Andezytowego",
+  "block.dustrial_decor.cast_iron_brick_slab": "Ceglana Płyta Żelaza Andezytowego",
+  "block.dustrial_decor.cast_iron_brick_vertical_slab": "Ceglana Pionowa Płyta Żelaza Andezytowego",
+  "block.dustrial_decor.cast_iron_brick_stairs": "Ceglane Schody Żelaza Andezytowego",
+  "block.dustrial_decor.engraved_cast_iron_block": "Grawerowany Blok Żelaza Andezytowego",
+  "block.dustrial_decor.cast_iron_balustrade": "Balustrada Żelaza Andezytowego"
+}
```

Comment:
First of all: Andesite Iron is a material from which the blocks are made out of, so the correct translation would be "&lt;x:&gt; z żelaza andezytowego".

Second of all: "Billet" would be better translated as "sztabka".

Reply:
sztabka is already used by all ingots like Sztabka żelaza

Reply:
Billets are pretty similar to ingots - see this:
https://blog.thepipingmart.com/other/ingot-vs-billet-whats-the-difference/

- Oh, nvm, my bad, it's translated to "Kęs" apparently...